packages/neuron-wallet/src/utils/rpc-request.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios'
+
+export const rpcBatchRequest = async (
+  ckbIndexerUrl: string,
+  options: {
+    method: string
+    params?: any
+  }[]
+): Promise<any[]> => {
+  const res = await axios({
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json'
+    },
+    data: options.map(v => ({
+      id: 0,
```

Comment:
`id` should be different in the array, it is used to correlate request and response among the request array and response array.